src/Components/Nav/Nav.js

```diff
@@ -1,7 +1,241 @@
+          <NavMain>
+            <div>
+              <Link to="/">
+                <MainLogo alt="오늘의 집" src="/images/logo.png" />
+              </Link>
+            </div>
+            <NavMainContent>
+              <Link
+                onMouseEnter={handleCategoryOn}
+                onMouseLeave={handleCategoryOff}
+                className="mainContent-menu"
+                to="/"
+              >
+                커뮤니티
+              </Link>
+            </NavMainContent>
+            <NavMainContent>
+              <Link className="mainContent-menu" to="/">
+                스토어
+              </Link>
+            </NavMainContent>
+            <NavMainContent>
+              <Link className="mainContent-menu" to="/">
+                인테리어 시공
+              </Link>
+            </NavMainContent>
+          </NavMain>
+          <NavMain>
+            <div>
+              <form className="form">
+                <input placeholder="스토어검색" type="text" required />
+                <button type="submit">
+                  <BiSearchs className="inputIcon"></BiSearchs>
+                </button>
+                <button type="submit">
+                  <MdPhotoCameras></MdPhotoCameras>
+                </button>
+              </form>
+            </div>
+            <div>
+              <Link to="/">
+                <BsBookmark className="Nav-Aside"></BsBookmark>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
+                <BsBell className="Nav-Aside"></BsBell>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
+                <BiCart className="Nav-Aside"></BiCart>
+              </Link>
+            </div>
+            <div>
+              <Link to="/">
+                <CgProfile className="Nav-Aside"></CgProfile>
+              </Link>
+            </div>
+            <div>
+              <button className="writeBtn" type="button">
+                글쓰기
+                <AiOutlineDown className="AiOutlineDown"></AiOutlineDown>
+              </button>
+            </div>
+          </NavMain>
+        </NavBar>
+      </HeadNav>
+      <div>
+        <MainCategory>
+          {NAV_DATA.map(item => {
+            return (
+              <li>
+                <Link className="MainCategory_Link" to="/">
+                  {item}
+                </Link>
+              </li>
+            );
+          })}
+        </MainCategory>
+      </div>
+      <div>
+        {{ categoryShow } ? (
+          <Category
+            categoryShow={categoryShow}
+            handleCategoryOff={handleCategoryOff}
+          />
+        ) : (
+          ''
+        )}
+      </div>
+    </div>
+  );
 }
+const HeadNav = styled.header`
+  border-bottom: 1px solid #dbdbdb;
+`;
+const NavBar = styled.nav`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 80px;
+  margin: 0 7%;
+  padding: 10px 40px;
+`;
+const NavMain = styled.div`
```

Comment:
```suggestion
const HeadNav = styled.header`
  border-bottom: 1px solid #dbdbdb;
`;

const NavBar = styled.nav`
  display: flex;
  align-items: center;
  justify-content: space-between;
  height: 80px;
  margin: 0 7%;
  padding: 10px 40px;
`;

const NavMain = styled.div`
```
🤔 🤔 🤔 ...
반복적인 리뷰 안나오도록 애써주세요!

Reply:
띄어쓰기 이제 신경쓰고 하고있습니다 ㅠ